src/starknet_interface_generator/order_imports.py

```diff
@@ -0,0 +1,61 @@
+from multiprocessing import allow_connection_pickling
+from symbol import import_name
+from typing import List, Dict
+from collections import OrderedDict
+from starkware.cairo.lang.compiler.ast.code_elements import CodeBlock, CodeElementImport, CodeElementEmptyLine
+from starkware.cairo.lang.compiler.ast.visitor import Visitor
+
+class OrderImports(Visitor):
+    """
+    Orders imports in Cairo files
+    """
+
+    def __init__(self, import_order_names: List[str]):
+        super().__init__()
+        self.import_order_names=import_order_names
+
+    def _visit_default(self, obj):
+        # top-level code is not generated
+        return obj
+    
+    def visit_CodeBlock(self, elm: CodeBlock):
+        return self.extract_imports(elm)
+    
+    def extract_imports(self, elm):
+        empty_element = self.get_empty_element(elm)
```

Comment:
This is not a reliable method to get all the imports in the file imo. What if they are not correctly formatted and blank lines are missing? This would lead to a lot of problems.

Reply:
This line is for detecting imports, it is just for getting empty line for later formatting of imports, so we can separate import groups with empty line in final file. But you right, I should generate empty line regardless if is present in the file